Tools/scripts/patchcheck.py

```diff
@@ -6,9 +6,13 @@
 import os.path
 import subprocess
 import sysconfig
+import functools
 
 import reindent
 import untabify
+from io import StringIO
+from test.support import change_cwd
+from duplicate_meth_defs import main as find_duplicates
```

Comment:
is it a tool or a library?  We're only using it as a library.  Lets design it as a library instead of treating main as an API with a stubbed out stdout.